index.html

```diff
@@ -0,0 +1,73 @@
+                <a>Mac</a>
+                <a>iPod</a>
+                <a>iPhone</a>
+                <a>iPad</a>
+                <a>iTunes</a>
+                <a>Support</a>
+                <div class="search-bar">
+                    <img class="search-icon-white" alt="svgImg"
+                        src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHg9IjBweCIgeT0iMHB4Igp3aWR0aD0iMjQiIGhlaWdodD0iMjQiCnZpZXdCb3g9IjAgMCAxNzIgMTcyIgpzdHlsZT0iIGZpbGw6IzAwMDAwMDsiPjxnIGZpbGw9Im5vbmUiIGZpbGwtcnVsZT0ibm9uemVybyIgc3Ryb2tlPSJub25lIiBzdHJva2Utd2lkdGg9IjEiIHN0cm9rZS1saW5lY2FwPSJidXR0IiBzdHJva2UtbGluZWpvaW49Im1pdGVyIiBzdHJva2UtbWl0ZXJsaW1pdD0iMTAiIHN0cm9rZS1kYXNoYXJyYXk9IiIgc3Ryb2tlLWRhc2hvZmZzZXQ9IjAiIGZvbnQtZmFtaWx5PSJub25lIiBmb250LXdlaWdodD0ibm9uZSIgZm9udC1zaXplPSJub25lIiB0ZXh0LWFuY2hvcj0ibm9uZSIgc3R5bGU9Im1peC1ibGVuZC1tb2RlOiBub3JtYWwiPjxwYXRoIGQ9Ik0wLDE3MnYtMTcyaDE3MnYxNzJ6IiBmaWxsPSJub25lIj48L3BhdGg+PGcgZmlsbD0iI2ZmZmZmZiI+PHBhdGggZD0iTTczLjQ1ODMzLDIxLjVjLTI4LjYxMDkyLDAgLTUxLjk1ODMzLDIzLjM0NzQzIC01MS45NTgzMyw1MS45NTgzM2MwLDI4LjYxMDkgMjMuMzQ3NDEsNTEuOTU4MzMgNTEuOTU4MzMsNTEuOTU4MzNjMTEuNzI4ODIsMCAyMi41MzUyOSwtMy45NjY2OSAzMS4yNDkxOSwtMTAuNTY4MDNsMzMuNTU4NzYsMzMuNTUxNzZjMS43OTc1MiwxLjg3MjIzIDQuNDY2NzUsMi42MjY0MSA2Ljk3ODI1LDEuOTcxNjhjMi41MTE1LC0wLjY1NDcyIDQuNDcyODIsLTIuNjE2MDUgNS4xMjc1NSwtNS4xMjc1NWMwLjY1NDcyLC0yLjUxMTUgLTAuMDk5NDYsLTUuMTgwNzMgLTEuOTcxNjgsLTYuOTc4MjVsLTMzLjU1MTc2LC0zMy41NTg3NmM2LjYwMTM1LC04LjcxMzkgMTAuNTY4MDMsLTE5LjUyMDM4IDEwLjU2ODAzLC0zMS4yNDkxOWMwLC0yOC42MTA5IC0yMy4zNDc0MSwtNTEuOTU4MzMgLTUxLjk1ODMzLC01MS45NTgzM3pNNzMuNDU4MzMsMzUuODMzMzNjMjAuODY0NjIsMCAzNy42MjUsMTYuNzYwMzkgMzcuNjI1LDM3LjYyNWMwLDEwLjAxNzk2IC0zLjg5MTI3LDE5LjA2NTQ5IC0xMC4yMTgxLDI1Ljc5MDJjLTAuNjE1ODYsMC40NDU4NSAtMS4xNTY4NiwwLjk4Njg0IC0xLjYwMjcsMS42MDI3Yy02LjcyNjE2LDYuMzM0ODcgLTE1Ljc3OTA3LDEwLjIzMjEgLTI1LjgwNDIsMTAuMjMyMWMtMjAuODY0NjIsMCAtMzcuNjI1LC0xNi43NjAzOSAtMzcuNjI1LC0zNy42MjVjMCwtMjAuODY0NjEgMTYuNzYwMzgsLTM3LjYyNSAzNy42MjUsLTM3LjYyNXoiPjwvcGF0aD48L2c+PC9nPjwvc3ZnPg==" />
+                    <input class="search-box" type="text" name="search" />
+                </div>
+            </div>
+        </header>
+        <section class="main-content">
+            <div class="main-text">
+                <a href=""><img class="main-text1" src="./assets/main-img1.png" alt="textimage"></a>
+                <a href=""><img src="./assets/main-img2.png" alt="textimage1"></a>
+                <a href=""><img class="main-text3" src="./assets/main-text3.png" alt="textimage2"></a>
+            </div>
+        </section>
+        <section class='bottom-grid'>
+            <img src="./assets/bottom-img.jpg" alt="bottomimage1">
+            <img src="./assets/bottom-img2.jpg" alt="bottomimage2">
+            <img src="./assets/bottom-img1.jpg" alt="bottomimage3">
+            <img src="./assets/bottom-img4.jpg" alt="bottomimage4">
+        </section>
+        <footer>
+            <section class="footer-nav">
```

Comment:
- [ ] The class could be assigned to the `<footer>` this way the code will be DRYer, as it won't have unnecessary containers.

Reply:
Thanks for the Info, Made changes accordingly.